web/src/main/java/nl/tehdreamteam/se42/web/soap/conversation/SoapConversationEndpoint.java

```diff
@@ -14,19 +16,24 @@
  * @author Oscar de Leeuw
  */
 @WebService(targetNamespace = SoapWebServiceConstants.DEFAULT_NAMESPACE + "conversation")
-public class SoapConversationController implements ConversationController {
+public class SoapConversationEndpoint implements ConversationEndpoint {
+
+    private static final Logger logger = LogManager.getLogger(SoapConversationEndpoint.class.getSimpleName());
+
     @Override
     public void addMessage(@WebParam(name = "token") String token, @WebParam(name = "content") String content, @WebParam(name = "conversation_id") long conversationId) {
-
+        logger.debug("Soap request for adding a message (token='{}', content='{}', conversation_id='{}').", token, content, conversationId);
     }
 
     @Override
     public void addUserToConversation(@WebParam(name = "token") String token, @WebParam(name = "user_id") long userId, @WebParam(name = "conversation_id") long conversationId) {
-
+        logger.debug("Soap request for adding an user to a conversation (token='{}', user_id='{}', conversation_id='{}').", token, userId, conversationId);
     }
 
     @Override
     public List<Message> getMessages(@WebParam(name = "token") String token, @WebParam(name = "conversation_id") long conversationId) {
+        logger.debug("Soap request for getting the messages of a conversation (token='{}', conversation_id='{}').", token, conversationId);
+
         return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1168)